doc/user_guide/psyir.rst

```diff
@@ -219,6 +220,28 @@ tree from an ancestor node to the node:
 
 .. automethod:: psyclone.psyir.nodes.Node.path_from
 
+Tree Interrogation
+==================
+
+Each PSyIR node provides several ways to interrogate the AST:
+
+Following the `parent` and `children` terminology, we define a node's `siblings`
+as the children of its parents. Note that this definition implies that all nodes
```

Comment:
As I've returned this to you, would you also mind replacing 'parents' with 'parent' here (as a node only has a single parent) and you get the glory (?)  of being able to add your name to the author list (conf.py).

Reply:
Done, see 2f1afecb0.

Reply:
Sorry @arporter, which part of the `conf.py` file should I add to? There seem to be three different author lists.

Reply:
Oh yes, all three of them please!

Reply:
Oh actually there were another two. I think this were the right places? 15c0b1a13